data/ingredient_taxonomy.json

```diff
@@ -72,23 +73,23 @@
     },
     "carrots": {
       "ingredient_name": "Carottes",
-      "ingredient_id": "onions",
+      "ingredient_id": "carrots",
       "category_id": "vegetables",
       "quantities": [
         "carrot"
       ]
     },
     "courgettes": {
       "ingredient_name": "Courgettes",
-      "ingredient_id": "carrots",
+      "ingredient_id": "courgettes",
```

Comment:
why are some vegetables named in English and others not?

Reply:
All the data come from this [spreadsheet](https://docs.google.com/spreadsheets/d/1aqYX47vkJnjiKVqLku_-f8IlbiFkjwpkgFyGGTSDfdI/edit?usp=drive_web&ouid=105092150874234810207) so probably stephan wrote some code between two edits and so switch to english 🤷

Personally, I don't care at all about the content since it's "fake" data. Cleaner one will be created with nutritioniste.

Reply:
ah ok